sklearn/metrics/_scorer.py

```diff
@@ -758,7 +759,7 @@ def make_scorer(
     median_absolute_error, greater_is_better=False
 )
 neg_root_mean_squared_error_scorer = make_scorer(
```

Comment:
Let's add the `neg_root_mean_squared_log_error_scorer`.
We also need to mention it in the entry of the changelog.